### PR TITLE
New package: Evdial.MikuMod version 21

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/113164&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/e/Evdial/MikuMod/21/Evdial.MikuMod.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Evdial.MikuMod
+PackageVersion: 21
+PackageLocale: en-US
+Publisher: EvDial
+PackageName: Hatsune Miku Modification
```
Comment (stephengillie) [inline manifests/e/Evdial/MikuMod/21/Evdial.MikuMod.locale.en-US.yaml:8]: Please add a PackageUrl.

File: manifests/e/Evdial/MikuMod/21/Evdial.MikuMod.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Evdial.MikuMod
+PackageVersion: 21
+PackageLocale: en-US
+Publisher: EvDial
+PackageName: Hatsune Miku Modification
+License: Apache 3
+ShortDescription: 3 original songs, 6 bonus songs, cutscenes, cool menu shit
```
Comment (Trenly) [inline manifests/e/Evdial/MikuMod/21/Evdial.MikuMod.locale.en-US.yaml:10]:
```suggestion
ShortDescription: 3 original songs, 6 bonus songs, cutscenes, cool menu
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/e/Evdial/MikuMod/21/Evdial.MikuMod.locale.en-US.yaml (hunk 2 above)
Comment (upintheairsheep) [reply manifests/e/Evdial/MikuMod/21/Evdial.MikuMod.locale.en-US.yaml]: this appears to be the official description, not one I made

File: manifests/e/Evdial/MikuMod/21/Evdial.MikuMod.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Evdial.MikuMod
+PackageVersion: 21
+PackageLocale: en-US
+Publisher: EvDial
+PackageName: Hatsune Miku Modification
+PackageUrl: https://gamebanana.com/mods/44307
+License: Apache 3
```
Comment (Trenly) [inline manifests/e/Evdial/MikuMod/21/Evdial.MikuMod.locale.en-US.yaml:10]:
```suggestion
License: CC-BY-NC-ND-4.0
LicenseUrl: https://gamebanana.com/mods/44307
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
This package looks like a mod for another game - should that game be a dependency? 
image

**Comment by upintheairsheep** [comment]
> This package looks like a mod for another game - should that game be a dependency?
> image

https://gamebanana.com/questions/21214

**Comment by upintheairsheep** [comment]
Requires FileDaddy, dependancy

**Comment by stephengillie** [comment]
> Requires FileDaddy, dependancy

This package isn't currently in the WinGet public manifests, so there is an opportunity to add it as well.

**Comment by Trenly** [comment]
Dependencies are now supported in the 1.6 client. If you could please submit a new manifest for the latest version of the package, it would be greatly appreciated!

**Comment by stephengillie** [comment]
> > Requires FileDaddy, dependancy
> 
> This package isn't currently in the WinGet public manifests, so there is an opportunity to add it as well.

This is unchanged.

```PowerShell
PS C:\ManVal> Find-WinGetPackage FileDaddy
PS C:\ManVal>
```

**Pull request closed without merging**